src/edu/wright/cs/raiderplanner/content/IMAGE-CREDITS.md

```diff
@@ -7,4 +7,4 @@ settings_about.png - GIMP image, created by Clayton D. Terrill
 settings_account.png - GIMP image, created by Clayton D. Terrill
 settings_general.png - GIMP image, created by Clayton D. Terrill
 settings_goBack.png - GIMP image, created by Clayton D. Terrill
-settings_theme.png - GIMP image, created by Clayton D. Terrill
\ No newline at end of file
+settings_theme.png - GIMP image, created by Clayton D. Terrill, created by Hayden M. Westfall information.png - https://www.flaticon.com/
```

Comment:
This change does not make sense.  You should have `information.png` on a new line and then just the (complete) URL, like this:
```
information.png - https://www.example.com/path/to/some/image.png
```
Please fix this.